src/transforms/code-scanning-alert.ts

```diff
@@ -1,35 +1,80 @@
 import issueKeyParser from "jira-issue-key-parser";
-import { Context } from "probot/lib/context";
-import { JiraRemoteLinkData } from "../interfaces/jira";
+import {Context} from "probot/lib/context";
+import {JiraRemoteLinkData} from "../interfaces/jira";
 
-function getLastUpdated({ status, started_at, completed_at }) {
-	if (status === 'completed') {
-		return completed_at;
-	} else {
-		return started_at;
+// TODO ARC-618
+const getPullRequestTitle = (id: number): string => {
+	return `GH-${id}`;
+}
+
+const getEntityTitle = (ref: string): string => {
+	// ref can either be a branch reference or a PR reference
+	const components = ref.split("/")
+	switch (components[1]) {
+		case "heads": // branch
+			// The branch name may contain forward slashes! Rejoin them
+			return components.slice(2).join("/");
+		case "pull": // pull request
+			return getPullRequestTitle(parseInt(components[2]));
+		default:
+			// log something here
+			return "";
+	}
+}
+
+const transformStatusToAppearance = (status: string): string => {
```

Comment:
Can we add a comment explaining possible values of `status` and valid values for `appearance` please? Similar to what was done [here](https://github.com/atlassian/github-for-jira/blob/main/src/transforms/deployment.ts#L11).

Reply:
Good idea. I might take this a step further and define an enum type for the `appearance` (like Jira has)